docs/src/app/components/AppLeftNav.jsx

```diff
@@ -69,7 +69,20 @@ const AppLeftNav = React.createClass({
   },
 
   handleVersionChange: function(event, index, value) {
-    window.location = value;
+    if (value === this.firstNonPreReleaseVersion()) {
+      window.location = 'http://www.material-ui.com/';
```

Comment:
That was by design: 

> The second makes the docs menu show HEAD for local installs, and lets you open the past versions on the material-ui.com site.

It seemed preferable to have the version menu work on a local install. There are plenty of workarounds if you need to open a gh-pages copy.